js-loops/js-loops_forof-list/js/index.js

```diff
@@ -17,6 +30,8 @@ const programmingLanguages = [
   "Ruby",
 ];
 
-// --v-- write/change code here --v--
-
-// --^-- write/change code here --^--
+for (const i of programmingLanguages) {
```

Comment:
Wouldn't it be better to use a let here? Because it changes constantly? I am not sure - if it works it's fine!